Src/BootCamp.Chapter/Settings.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Globalization;
+
+namespace BootCamp.Chapter
+{
+    public static class Settings
+    {
+        public static NumberFormatInfo numberFormatInfo = new NumberFormatInfo { NumberDecimalSeparator = "." };
+        public static CultureInfo cultureInfo = new CultureInfo("en-GB");
+        public static char divider = ',';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5054323)